src/SfxWeb/src/app/modules/event-store/event-store-timeline/event-store-timeline.component.ts

```diff
@@ -44,7 +44,9 @@ export class EventStoreTimelineComponent implements AfterViewInit, OnChanges {
     let items = new DataSet<DataItem>();
   
     // create visualization
-    this._timeline = new Timeline(this.container.nativeElement, items, groups);
+    this._timeline = new Timeline(this.container.nativeElement, items, groups, {
+        locale: 'en_US'
```

Comment:
This is hardcoding it to english? Can't we read the locale of the user?

Reply:
We could read the user's locale but it makes for a weird experience if only one table in all of SFX localizes like that, id rather have a point where localization is handled properly for a much wider amount of SFX.